src/bootstrap/src/core/sanity.rs

```diff
@@ -99,6 +99,22 @@ pub fn check(build: &mut Build) {
         cmd_finder.must_have("git");
     }
 
+    if !build.config.dry_run() && build.config.llvm_from_ci {
+        let cxx = build.cxx(build.build).unwrap();
+        let mut cxx_cmd = Command::new(cxx);
+        cxx_cmd.arg("-dumpversion");
```

Comment:
Isn't this checking the compiler version rather than libstdcxx version? These are fairly likely to be the same if the compiler is gcc, but not otherwise.

Reply:
I thought it was the case for any c++ compiler https://github.com/rust-lang/rust/blob/1c38c466de58b9d619d01f0790a61f6ef8c71d42/src/bootstrap/src/core/sanity.rs#L112

I have to check if there is a better way.

Reply:
Should we check libc++ version too? If so, what would be a good threshold for it?